app.rb

```diff
@@ -4,9 +4,11 @@
 require './book'
 require './classroom'
 require './rental'
+require './print_script'
 
-class App
+class App < PrintScript
```

Comment:
- [ ] Please, kindly include both authors on your `readme.md` file.  That way, both authors will take credit for this magnificent job!